connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMakerConfig.java

```diff
@@ -199,6 +199,7 @@ public Map<String, String> workerConfig(SourceAndTarget sourceAndTarget) {
         props.putAll(stringsWithPrefix(CONFIG_PROVIDERS_CONFIG));
 
         // fill in reasonable defaults
+        props.putIfAbsent(CommonClientConfigs.CLIENT_ID_CONFIG, sourceAndTarget.toString());
```

Comment:
What does this change? I haven't looked too closely but I haven't seen a place where this will have any effect yet.

Reply:
Will remove the change from MirrorMaker, then this will start to make sense, thanks for catching this